src/frontend/packages/core/src/shared/components/page-header/page-header.component.ts

```diff
@@ -202,6 +223,22 @@ export class PageHeaderComponent implements OnDestroy, AfterViewInit {
       map(noLogout => !noLogout)
     );
 
+    this.tokenEnvelope = this.getUAAToken() 
+
+    this.authToken$ = tokenEnvelope
```

Comment:
You forgot to change it here and below in line 233 and 238 :)  
```suggestion
    this.authToken$ = this.tokenEnvelope
```